src/components/entity/data.vue

```diff
@@ -10,11 +10,17 @@ including this file, may be copied, modified, propagated, or distributed
 except according to the terms contained in the LICENSE file.
 -->
 <template>
-  <page-section v-if="dataset.dataExists && dataset.properties.length !== 0"
```

Comment:
We now always render this section, even if the dataset doesn't have properties. While the request for the entity is in progress, `EntityShow` hides this component. Once the response is received, if the request for the dataset is still in progress, the section title and a loading message is shown. Once that response is received, the edit button is shown (unless the user is a project viewer), and the entity data is shown (unless the dataset has no properties). If the dataset has no properties, then there will just be the section title and the button. If the dataset has no properties, **and** the user is a project viewer, they'll just see the section title. I don't think we need special logic for that case though: I don't think it's especially likely, and we're thinking of moving the entity data elsewhere at some point.